Cargo.toml

```diff
@@ -17,6 +17,7 @@ build = "build.rs"
 libc = "0.2"
 nom = { version = "2", features = ["verbose-errors"] }
 byteorder = "0.5"
+time = "0.1"
```

Comment:
The only reason I used the time crate is because of the "unlimited" value the limit can have. I used -1 to define that unlimited and that's not something that std::time::Duration allows (it doesn't allow negative values)

Maybe I could use a better type to express that value but I'm really not sure which one would be best. The kernel defines the "unlimited" as `RLIM_INFINITY` which equals to -1